src/main/java/io/fabric8/maven/docker/log/LogDispatcher.java

```diff
@@ -37,15 +40,16 @@ public LogDispatcher(DockerAccess dockerAccess) {
     }
 
     public synchronized void trackContainerLog(String containerId, LogOutputSpec spec) throws FileNotFoundException {
-        LogGetHandle handle = dockerAccess.getLogAsync(containerId, new DefaultLogCallback(spec));
+        LogGetHandle handle = dockerAccess.getLogAsync(containerId, createLogCallback(spec));
```

Comment:
I'm not a a fan of static imports as they dilute the real locations and I always search first in the current class when doing review. Could we refer to the static method via its class (`LogCallbackFactory.createLogBack()`) ? Even saves some lines.